### PR TITLE
Adding in all other acquia environments. Changing language to make it…

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**
… more clear we can deploy a branch

@doctoryes @macdiesel Please review.
FYI @edx/learner-growth

### CODE REVIEW

**Review by feanil — APPROVED** [review]
Seems good to me,  looks like the failure is linting issues.

**Comment by Dillon-Dumesnil** [comment]
This PR relates to https://github.com/edx/edx-gomatic/pull/372

**Review by doctoryes — APPROVED** [review]

**Comment by Dillon-Dumesnil** [comment]
@robrap

File: tubular/scripts/drupal_deploy.py [hunk 1]
```diff
@@ -20,18 +20,19 @@
...
+def deploy(env, username, password, branch_or_tag):
     """
     Deploys a given tag to the specified environment.
 
     Args:
         env (str): The environment to deploy code in (e.g. test or prod)
         username (str): The Acquia username necessary to run the command.
         password (str): The Acquia password necessary to run the command.
-        tag (str): The tag to deploy to the specified environment.
+        branch_or_tag (str): The branch or Ctag to deploy to the specified environment.
```
Comment (robrap) [inline tubular/scripts/drupal_deploy.py:33]: `Ctag` is a typo.  Not sure if this still lives.

Comment (Dillon-Dumesnil) [reply tubular/scripts/drupal_deploy.py:33]: Fixed in https://github.com/edx/tubular/pull/204

**Pull request merged**